test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/FileProviderRazorProjectTest.cs

```diff
@@ -188,13 +188,13 @@ public void EnumerateFiles_IteratesOverAllCshtmlUnderPath()
                 });
         }
 
-        [Fact(Skip = "Need to follow-up https://github.com/aspnet/Mvc/pull/7228")]
+        [Fact]
         public void GetItem_ReturnsFileFromDisk()
         {
             var fileProvider = new TestFileProvider("BasePath");
-            var file1 = fileProvider.AddFile("File1.cshtml", "content");
-            var file2 = fileProvider.AddFile("File2.js", "content");
-            var file3 = fileProvider.AddFile("File3.cshtml", "content");
+            var file1 = fileProvider.AddFile("/File1.cshtml", "content");
```

Comment:
Why do you need this change?

Reply:
The TestFileProvider.AddFile method actually accepts filepaths and not just file name. We we're using it interchangeably in multiple tests. So I just made sure we passed it in the same format as all the other tests